src/Microsoft.AspNetCore.Server.Kestrel/KestrelServer.cs

```diff
@@ -108,8 +114,45 @@ public void Start<TContext>(IHttpApplication<TContext> application)
                     else
                     {
                         atLeastOneListener = true;
-                        _disposables.Push(engine.CreateServer(
-                            parsedAddress));
+
+                        if (parsedAddress.Host.Equals("localhost", StringComparison.OrdinalIgnoreCase))
+                        {
+                            var exceptions = new List<Exception>();
```

Comment:
Yeah that was different before and I didn't revert this. I was throwing an `AggregateException`.